airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/ctid/PostgresCtidHandler.java

```diff
@@ -53,14 +56,18 @@ public class PostgresCtidHandler {
   private final String quoteString;
   private final CtidStateManager ctidStateManager;
   private final Map<AirbyteStreamNameNamespacePair, Long> fileNodes;
+  final Map<AirbyteStreamNameNamespacePair, TableBlockSize> tableBlockSizes;
   private final Function<AirbyteStreamNameNamespacePair, JsonNode> streamStateForIncrementalRunSupplier;
   private final BiFunction<AirbyteStreamNameNamespacePair, JsonNode, AirbyteStateMessage> finalStateMessageSupplier;
+  private static final int QUERY_TARGET_SIZE_GB = 1; // TODO: find optimal size
```

Comment:
small nit : can we create a tracking work item to find the correct size?

Reply:
- [ ] Investigate sync parameters for ctid with large tables.

Reply:
#27787